### PR TITLE
Add azure cloud module playbook

### PR DESCRIPTION
**Pull request by elyezer**

### CODE REVIEW

**Comment by AlanCoding** [comment]
might want to name it azure_rm instead, you'll see this pattern in many other places.

**Comment by elyezer** [comment]
@AlanCoding just updated the name of the playbook. Thanks for the suggestion.

File: cloud_modules/azure_rm.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- name: Create and remove an Azure virtual machine
+  hosts: localhost
+  connection: local
+  gather_facts: False
+
+  tasks:
+    - name: Create VM
+      azure_rm_virtualmachine:
+        admin_username: "{{ admin_username | default('tower-qe') }}"
+        admin_password: "{{ admin_password | default('YouSh@llNotPas5') }}"
```
Comment (kdelee) [inline cloud_modules/azure_rm.yml]: this is a public repo maybe we should not put this info here or generate something random

**Comment by elyezer** [comment]
Talked to @kdelee offline and we will leave this as is and update if needed.

**Pull request merged**